fala/assets-src/scripts/find-legal-adviser.js

```diff
@@ -253,9 +253,9 @@ exports.FindLegalAdviser = {
     var image;
     options = options || {};
 
-    if(options.icon) {
+    if(options.icon && options.icon==='icon-location-2x') {
```

Comment:
_not a blocker_

I wonder if there's any circumstance we call _without_ `icon`? "icon" may be an unnecessary configuration and we could default to `icon-location-2x`, unless `id` is given.